clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/HelixParticipationService.java

```diff
@@ -360,6 +369,27 @@ private void asyncStart() {
     });
   }
 
+  static void resetAllInstanceCVStates(
+      HelixPartitionStatusAccessor accessor,
+      StorageService storageService,
+      Logger currentLogger) {
+    // Get all hosted stores
+    currentLogger.info("Started resetting all instance CV states");
+    Map<String, Set<Integer>> storePartitionMapping = storageService.getStoreAndUserPartitionsMapping();
+    storePartitionMapping.forEach((storeName, partitionIds) -> {
+      partitionIds.forEach(partitionId -> {
+        try {
+          accessor.deleteReplicaStatus(storeName, partitionId);
+          currentLogger.info("Deleting CV state for resource: {} and partition id: {}", storeName, partitionId);
```

Comment:
Nitpick: Can we do this once for all partitions of the store (could log the `partitionIds ` set in a single line if you want to know the exact partition numbers) rather than one log line per partition? Even at one log per store it could still be quite noisy on the multi-tenant clusters...

Reply:
Sure, I can do that.